### PR TITLE
Add DockerCon promo code info

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Advertise DockerCon promo codes for new customers.

### CODE REVIEW

**Review by jerae-duffin — DISMISSED** [review]
LGTM

File: _includes/landing-page/dockercon-banner.html [hunk 1]
```diff
@@ -6,6 +6,9 @@
         <h2>
           DockerCon Live is coming this May 27th!
         </h2>
+      </br>
+      <p><b>Sign up for a Pro or a Team account today and save 20%</b></p>
+      <p>If you are a new or a returning customer, use the <b>DockerCon21</b> promo code to receive a 20% discount on an annual subscription, or for the first 3 months on a monthly subscription. <a href="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade" target="_blank">Sign up today</a>.</p>
```
Comment (olegburov) [inline _includes/landing-page/dockercon-banner.html:11]: `DockerCon21` will work too, but we voted for all upper case. See here:  https://docker.slack.com/archives/C022MELTG76/p1621632333045000?thread_ts=1621631402.044600&cid=C022MELTG76

```suggestion
      <p>If you are a new or a returning customer, use the <b>DOCKERCON21</b> promo code to receive a 20% discount on an annual subscription, or for the first 3 months on a monthly subscription. <a href="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade" target="_blank">Sign up today</a>.</p>
```

File: _includes/landing-page/dockercon-banner.html [hunk 2]
```diff
@@ -6,6 +6,9 @@
         <h2>
           DockerCon Live is coming this May 27th!
         </h2>
+      </br>
+      <p><b>Sign up for a Pro or a Team account today and save 20%</b></p>
```
Comment (olegburov) [inline _includes/landing-page/dockercon-banner.html:10]: There are user or organization accounts and Pro and Team (billing) plans.

```suggestion
      <p><b>Sign up for a Pro or a Team plan today and save 20%</b></p>
```

**Review by olegburov — APPROVED** [review]
Left a couple of comments, but looks good to me 👍🏻

File: _includes/landing-page/dockercon-banner.html (hunk 1 above)
Comment (usha-mandya) [reply _includes/landing-page/dockercon-banner.html]: Thanks Oleg! Updated

File: _includes/landing-page/dockercon-banner.html (hunk 2 above)
Comment (usha-mandya) [reply _includes/landing-page/dockercon-banner.html]: I've updated it to subscriptions as per https://www.docker.com/dockercon/promo/

**Review by mkvitnitsky — APPROVED** [review]

**Pull request merged**